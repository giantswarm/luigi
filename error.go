package main

import "github.com/giantswarm/microerror"

var jsonObjectParseError = &microerror.Error{
	Kind: "jsonObjectParseError",
}

// IsJSONObjectParse asserts jsonObjectParseError.
func IsJSONObjectParse(err error) bool {
	return microerror.Cause(err) == jsonObjectParseError
}
