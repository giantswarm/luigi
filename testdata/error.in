{"caller":"github.com/giantswarm/micrologger@v0.3.1/activation_logger.go:93","level":"error","stack":{"annotation":"execution failed error: deployment failed (status: `FAILED`, reason: `Release \"aws-operator-k8scloudconfig-0.6.3\" failed: admission webhook \"validation.gatekeeper.sh\" denied the request: [denied by unique-operator-version] label app.kubernetes.io/version=8.7.1-dev is not unique in resources of kind \"Deployment\"`)","kind":"unknown","stack":[{"file":"/Users/kopiczko/go/src/github.com/giantswarm/opsctl/pkg/cmd/deploy/appdeploy/deployer.go","line":338},{"file":"/Users/kopiczko/go/pkg/mod/github.com/giantswarm/backoff@v0.2.0/retry.go","line":13},{"file":"/Users/kopiczko/go/src/github.com/giantswarm/opsctl/pkg/cmd/deploy/appdeploy/deployer.go","line":347},{"file":"/Users/kopiczko/go/src/github.com/giantswarm/opsctl/command/deploy/command.go","line":346},{"file":"/Users/kopiczko/go/src/github.com/giantswarm/opsctl/pkg/output/funcs.go","line":33},{"file":"/Users/kopiczko/go/src/github.com/giantswarm/opsctl/command/deploy/command.go","line":356}]},"time":"2020-07-01T12:21:29.465397+00:00","verbosity":0}
