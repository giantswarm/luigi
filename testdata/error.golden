E 07/01 12:21:29 | micrologger@v0.3.1/activation_logger.go:93 | verbosity=0
	/Users/kopiczko/go/src/github.com/giantswarm/opsctl/command/deploy/command.go:356
	/Users/kopiczko/go/src/github.com/giantswarm/opsctl/pkg/output/funcs.go:33
	/Users/kopiczko/go/src/github.com/giantswarm/opsctl/command/deploy/command.go:346
	/Users/kopiczko/go/src/github.com/giantswarm/opsctl/pkg/cmd/deploy/appdeploy/deployer.go:347
	/Users/kopiczko/go/pkg/mod/github.com/giantswarm/backoff@v0.2.0/retry.go:13
	/Users/kopiczko/go/src/github.com/giantswarm/opsctl/pkg/cmd/deploy/appdeploy/deployer.go:338
	unknown: execution failed error: deployment failed (status: `FAILED`, reason: `Release "aws-operator-k8scloudconfig-0.6.3" failed: admission webhook "validation.gatekeeper.sh" denied the request: [denied by unique-operator-version] label app.kubernetes.io/version=8.7.1-dev is not unique in resources of kind "Deployment"`)
